### ISSUE TITLE
Tostools since to have problem with number dumps in ies files

### ISSUE THREAD

**Issue by YamashitaRen** [Ell/tostools#2]
Hi, I tried to dump the item_Equips.ies in ies.ipf from iCBT2 but output seems awfully wrong and lacking.
(Compare with IPF Suite)
Same problem with the way shorter item_equip_classtype.ies

I think it's only bugging when it has to process numbers...

Thanks for the tool anyway. The csv output will be perfect for my usage :) (In fact I'd like to use it asap but well... :D)

**Comment by TwoLaid** [comment]
Hi, I'll see what I can do. Could you please upload your ies file somewhere ?

**Comment by YamashitaRen** [comment]
Yes, here it is : http://www.mediafire.com/download/sr1udxf5aeyoepd/extractedies.7z
item_equip_classtype.ies has only the first column wrong. item_Equips.ies has many wrong columns.
The wrong column seem to only be the one containing numbers.

Thanks for your reactivity :)

**Comment by TwoLaid** [comment]
Thats interesting.. because it's working for me. Can you please tell me more about the setup you are running? 
Windows/Linux/Mac 32/64bit? Python 2.7/3  32bit/64bit ?

**Comment by YamashitaRen** [comment]
Indeed.
I used the release exe, on Win7 64bit with hum... python --version giving me 2.7.9.
But usually I'm more a linux guy. So I'll try to use it there :)

**Comment by TwoLaid** [comment]
Okay please give it a try on Linux. Otherwise I'll test it on my Windows
machine tonight

**Comment by YamashitaRen** [comment]
Same problem on Archlinux.
And I think you had it too : https://github.com/Ell/tostools/blob/master/dumps/item_Equip.ies.csv
Look at all theses zero :/

**Comment by TwoLaid** [comment]
Hah, oh god I'm so sorry. I confused this repository with mine! ( https://github.com/TwoLaid/treeofsavior-tools ). I'm not sure why I'm still subscribed to this one. The structures are outdated.
If Python is an option for you, I urge you to checkout my repo :p
Otherwise, I can help you with this one, sorry again!

**Comment by YamashitaRen** [comment]
Ahahah, of course I'm okay with Python !
Will have a look to yours, thanks :)
